examples/train_and_evaluate_custom_model.ipynb

```diff
@@ -0,0 +1,1996 @@
+      "            \"0, 'warm_start': False}\\n\"\n",
+      "            '[2020-02-20 15:30:50 +0000] [8] [INFO] [classifier] 0.909 for '\n",
+      "            \"{'bootstrap': True, 'class_weight': None, 'criterion': 'gini', \"\n",
+      "            \"'max_depth': None, 'max_features': 'auto', 'max_leaf_nodes': \"\n",
+      "            \"None, 'min_impurity_decrease': 0.0, 'min_samples_leaf': 5, \"\n",
+      "            \"'min_samples_split': 10, 'min_weight_fraction_leaf': 0.0, \"\n",
+      "            \"'n_estimators': 256, 'n_jobs': 8, 'oob_score': False, 'verbose': \"\n",
+      "            \"0, 'warm_start': False}\\n\"\n",
+      "            '[2020-02-20 15:30:50 +0000] [8] [INFO] [classifier] 0.909 for '\n",
+      "            \"{'bootstrap': True, 'class_weight': None, 'criterion': 'gini', \"\n",
+      "            \"'max_depth': None, 'max_features': 'auto', 'max_leaf_nodes': \"\n",
+      "            \"None, 'min_impurity_decrease': 0.0, 'min_samples_leaf': 5, \"\n",
+      "            \"'min_samples_split': 10, 'min_weight_fraction_leaf': 0.0, \"\n",
+      "            \"'n_estimators': 512, 'n_jobs': 8, 'oob_score': False, 'verbose': \"\n",
+      "            \"0, 'warm_start': False}\\n\"\n",
+      "            '[2020-02-20 15:30:50 +0000] [8] [INFO] [classifier] 0.818 for '\n",
+      "            \"{'bootstrap': True, 'class_weight': None, 'criterion': 'gini', \"\n",
+      "            \"'max_depth': None, 'max_features': 'auto', 'max_leaf_nodes': \"\n",
+      "            \"None, 'min_impurity_decrease': 0.0, 'min_samples_leaf': 5, \"\n",
+      "            \"'min_samples_split': 20, 'min_weight_fraction_leaf': 0.0, \"\n",
+      "            \"'n_estimators': 32, 'n_jobs': 8, 'oob_score': False, 'verbose': \"\n",
+      "            \"0, 'warm_start': False}\\n\"\n",
+      "            '[2020-02-20 15:30:50 +0000] [8] [INFO] [classifier] 0.909 for '\n",
+      "            \"{'bootstrap': True, 'class_weight': None, 'criterion': 'gini', \"\n",
+      "            \"'max_depth': None, 'max_features': 'auto', 'max_leaf_nodes': \"\n",
+      "            \"None, 'min_impurity_decrease': 0.0, 'min_samples_leaf': 5, \"\n",
+      "            \"'min_samples_split': 20, 'min_weight_fraction_leaf': 0.0, \"\n",
+      "            \"'n_estimators': 64, 'n_jobs': 8, 'oob_score': False, 'verbose': \"\n",
+      "            \"0, 'warm_start': False}\\n\"\n",
+      "            '[2020-02-20 15:30:50 +0000] [8] [INFO] [classifier] 0.909 for '\n",
+      "            \"{'bootstrap': True, 'class_weight': None, 'criterion': 'gini', \"\n",
+      "            \"'max_depth': None, 'max_features': 'auto', 'max_leaf_nodes': \"\n",
+      "            \"None, 'min_impurity_decrease': 0.0, 'min_samples_leaf': 5, \"\n",
+      "            \"'min_samples_split': 20, 'min_weight_fraction_leaf': 0.0, \"\n",
+      "            \"'n_estimators': 128, 'n_jobs': 8, 'oob_score': False, 'verbose': \"\n",
+      "            
```

Comment:
We might want to remove the printouts from the code and increase the library logging level.

Reply:
Done